compiler/src/dotty/tools/dotc/core/Denotations.scala

```diff
@@ -1063,6 +1063,9 @@ object Denotations {
      */
     def showDcl(implicit ctx: Context): String = ctx.printer.dclText(this).show
 
+    /** Like `showDcl`, but use the initial denotation. */
+    def showInitialDcl(implicit ctx: Context): String = ctx.printer.dclText(initial).show
```

Comment:
I don't think we need that. Just replace `foo.showInitialDcl` with `foo.initial.showDcl`.